genre.rb

```diff
@@ -0,0 +1,14 @@
+class Genre
+  attr_accessor :name, :id, :items
```

Comment:
- please we don't need to access name and id here as a genre has its own id and name
- you should import item class to access it